docs/distributed/DESIGN.md

```diff
@@ -103,6 +88,40 @@ Input for the key is the object name specified in `PutObject()`, returns a uniqu
 
 - MinIO does erasure coding at the object level not at the volume level, unlike other object storage vendors. This allows applications to choose different storage class by setting `x-amz-storage-class=STANDARD/REDUCED_REDUNDANCY` for each object uploads so effectively utilizing the capacity of the cluster. Additionally these can also be enforced using IAM policies to make sure the client uploads with correct HTTP headers.
 
+- MinIO also supports expansion of existing clusters in zones, each zone is a self contained entity with same SLA's (read/write quorum) for each object as original cluster. By using the existing namespace for lookup validation MinIO ensures conflicting objects are not created, when no such object exists then MinIO simply uses the least used zone.
+
+*There are no limits on how many zones can be combined*
+
+```
+minio server http://host{1...32}/export{1...32} minio server http://host{5...6}/export{1...8}
```

Comment:
Is this correct?  Thought it should read `minio server http://host{1...32}/export{1...32} http://host{5...6}/export{1...8}`